src/libs/actions/ReimbursementAccount/resetFreePlanBankAccount.js

```diff
@@ -24,6 +24,17 @@ function resetFreePlanBankAccount(bankAccountID) {
         },
         {
             optimisticData: [
+                {
+                    onyxMethod: CONST.ONYX.METHOD.MERGE,
+                    key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+                    value: {
+                        shouldShowResetModal: false,
+                        isLoading: true,
+                        pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
```

Comment:
After a second thought, I think we should set `pendingAction` to `null` in both cases.

Reply:
On success, the `pendingAction` goes away because we replace the `reimbursementAccount` completely with this: https://github.com/Expensify/App/blob/0bd46f81f8f5dddc7c6dc8619cce05a94db85306/src/libs/actions/ReimbursementAccount/resetFreePlanBankAccount.js#L53-L57

Notice that it is not a MERGE, but a SET.

On failure, the `pendingAction`: I'm not sure... gonna ask in slack

Reply:
I asked in [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1677891548483909) and the agreement was to delete using `failureData` (as soon as the request fails and not when the user clicks the X to dismiss the error)

This is working like that ^ now.